src/common/styles-dictionary/css/variables.css

```diff
@@ -0,0 +1,230 @@
+/**
+ * Do not edit directly
+ * Generated on Mon, 15 Nov 2021 22:54:50 GMT
+ */
+
+:root {
+  --sds-font-body-xxxs-400-font-family: "Open Sans", "OpenSans-Regular";
```

Comment:
I was hoping to make the whole rule for each font token into a single variable that held all the values, but letter-spacing isn't part of the css `font` shorthand, so each property gets its own variable